### PR TITLE
Address review comments, update comment for _revertIfLupDroppedBelowL…

### PR DESCRIPTION
**Pull request by grandizzy**
…imit, changed LimitIndexReached to LimitIndexExceeded

### CODE REVIEW

**Review by dmitriia — APPROVED** [review]
Looks ok

File: src/libraries/external/BorrowerActions.sol [hunk 1]
```diff
@@ -201,10 +200,9 @@ library BorrowerActions {
             result_.t0PoolDebt += vars.t0DebtChange;
             result_.poolDebt   = Maths.wmul(result_.t0PoolDebt, poolState_.inflator);
 
-            // determine new lup index and revert if borrow happens at a price higher than the specified limit (lower index than lup index)
-            vars.lupId = _lupIndex(deposits_, result_.poolDebt);
```
Comment (EdNoepel) [inline src/libraries/external/BorrowerActions.sol:205]: Can we remove `lupId` from `DrawDebtLocalVars`?  Seems is not used anymore.

**Review by EdNoepel — COMMENTED** [review]
One comment inline.  Otherwise, looks good.

File: src/libraries/external/BorrowerActions.sol (hunk 1 above)
Comment (grandizzy) [reply src/libraries/external/BorrowerActions.sol:205]: ah, yes, I thought I did that already, let me change

Comment (grandizzy) [reply src/libraries/external/BorrowerActions.sol:205]: removed in https://github.com/ajna-finance/contracts/pull/607/commits/a707a28212d2e26549b5563359f9ffe612bbd134

**Review by EdNoepel — APPROVED** [review]
👌

**Review by dmitriia — COMMENTED** [review]
Ok

**Pull request merged**